inst/workflow/3_reporting/Groups.yaml

```diff
@@ -3,16 +3,16 @@ meta:
   ID: Groups
   Description: Reporting Groups Data
   Priority: 1
-spec: 
+spec:
   Mapped_STUDY:
     _all:
-      required: true
```

Comment:
were these meant to be removed? And does it need to be done on:

3_reporting/Bounds.yaml
4_modules/report_kri_country.yaml
4_modules/report_kri_site.yaml

inst/examples/2_AdverseEventWorkflow.R

?

Reply:
we went back and forth, but for the moment, we are leaving the `_all` specs as is, and will handle them later via #1952

Reply:
oh nice- thanks for catching that!